src/HeavenScroll.js

```
@@ -13,7 +13,8 @@ var defaultOptions = {
     urlQueryParamName: -1,
     loadPageFunction: function () {},
     spinnerClassName: 'Spinner',
-    debugMode: false
+    debugMode: false,
+    eraseMode: ''
```

Default mode is `''`, yet [here](https://github.com/Comum/jquery-heaven-scroll/blob/0653b2cf702f69aae2a45a3e44a6a4ae681b6114/src/HeavenScroll.js#L116) you define it as `hide` when it's empty. Please set the default value to `hide` and get rid of the `else` statement